.github/workflows/test_iree.yml

```diff
@@ -67,12 +67,49 @@ jobs:
           source ${VENV_DIR}/bin/activate
           pytest iree_tests/onnx/node/generated -n auto -rpfE --timeout=30 --retries 2 --retry-delay 5 --durations=10
 
+  linux_x86_64_w7900_gpu_models:
+    strategy:
+      matrix:
+        os: [nodai-amdgpu-w7900-x86-64]
+    name: Linux (x86_64 w7900) Models GPU
+    runs-on: ${{matrix.os}}
```

Comment:
Are you planning on expanding this matrix? As this is now, it's just making the names verbose and adding complexity to the yaml file.

* Matrix jobs are initially hidden under the group: https://github.com/nod-ai/SHARK-TestSuite/actions/runs/8546974154?pr=152 ![image](https://github.com/nod-ai/SHARK-TestSuite/assets/4010439/8c32c8d4-4e7c-4854-bcfa-0d9db60eb464)
* Long 'check' name: ![image](https://github.com/nod-ai/SHARK-TestSuite/assets/4010439/80c90a5e-fcc5-4b38-9cf3-060ef14d919e)